lib/ui/payment/stripe_subscription_page.dart

```diff
@@ -300,7 +304,8 @@ class _StripeSubscriptionPageState extends State<StripeSubscriptionPage> {
               if (isActive) {
                 return;
               }
-              if (!_isActiveStripeSubscriber &&
+              if (_currentSubscription.paymentProvider != kStripe &&
```

Comment:
Previously: When last strip subscription expires, the app was not allowing the user to change plans/re-subscribe and showing this dialog: 'please cancel your existing subscription from stripe first`